src/Microsoft.DotNet.Build.Tasks/PackageFiles/executor/config.json

```diff
@@ -0,0 +1,126 @@
+﻿{
```

Comment:
As this PR is more to receive feedback than to actually merge the code, for convenience I added the config.json file here.